src/lib.rs

```diff
@@ -37,18 +37,30 @@ The name of this library is inspired from the famous surrealist artist `Salvador
 
 extern crate nalgebra as na;
 extern crate num_traits as num;
-#[cfg(all(feature = "dim2", feature = "parry"))]
+#[cfg(all(feature = "dim2", feature = "f32"))]
```

Comment:
"parry" feature will still be needed if we want to keep salva optionally independent from parry.